Martium.TravelInfo.App/Forms/TravelInfoForm.cs

```diff
@@ -129,6 +141,7 @@ private void LoadTravelInfoSettings()
             DepartureAddressTextBox.Text = _travelInfoSettingsModel.DepartureAddress;
             PricePerKm.Text = _travelInfoSettingsModel.PricePerKm.ToString(CultureInfo.InvariantCulture);
             AdditionalDistanceInKmTextBox.Text = _travelInfoSettingsModel.AdditionalDistanceInKm.ToString(CultureInfo.InvariantCulture);
+            CountryIsoComboBox.Text = "LTU"; // if we choose to save departure country option then change this 
```

Comment:
read `value` from country dictionary using `_travelInfoSettingsModel.DepartureCountry` as key to search in dictionary

Reply:
Make `CountryIsoComboBox` **disabled** in `InitializeControls` method  after you will do testing (because in first stage we won't support multiple countries